lib/assets_server.rb

```diff
@@ -29,15 +35,23 @@ def self.get_local_svg(name, asset_type)
   get_local_asset 'layout/layout.html', 'lib/render/layout/layout.html', 'text/html'
   get_local_asset 'editor/editor.js', 'lib/render/editor/editor.js', 'application/javascript'
   get_local_asset 'editor/editor.css', 'lib/render/editor/editor.css', 'text/css'
-  get_local_asset 'editor/editor.html', 'lib/render/editor/editor.html', 'text/html'
   get_local_asset 'editor/hammer.min.js', 'lib/render/editor/hammer.min.js', 'application/javascript'
   get_local_asset 'boom.png', 'lib/public/boom.png', 'image/png'
 
+  get_media_assets 'media'
+  get_media_assets 'local-media'
+
   ['red', 'blue', 'green', 'black'].each do |name|
     get_local_svg(name, 'editor')
   end
 
   ['attires_enabled', 'attires_disabled'].each do |name|
     get_local_svg(name, 'layout')
   end
+
+  get '/assets/editor/editor.html' do
+    cross_origin
+    @assets_url = "//#{request.host_with_port}/assets"
+    erb File.read('lib/render/editor/editor.html.erb')
```

Comment:
Does this has a big impact on performance?

Reply:
Don't really know how to measure it. :thinking: